packages/core/chrome/core-chrome-browser-internal/src/project_navigation/project_navigation_service.ts

```diff
@@ -219,18 +220,20 @@ export class ProjectNavigationService {
    * Initialize a "serverless style" navigation. For stateful deployments (not serverless), this
    * handler initialize one of the solution navigations registered.
    *
+   * @param id Id for the navigation tree definition
    * @param navTreeDefinition$ The navigation tree definition
    * @param location Optional location to use to detect the active node in the new navigation tree
```

Comment:
nit: Is `location` still used?

Reply:
Good catch, removed in https://github.com/elastic/kibana/pull/179270/commits/4a4e8f7d8c6cd7923bd44cb565d07ab10937e25b